hail/python/hail/methods/impex.py

```diff
@@ -2972,6 +2972,337 @@ def import_avro(paths, *, key=None, intervals=None):
+
+    info(f'import_gvs: checkpointing combined VDS to temporary directory before repartition')
+    combined = combined.checkpoint(checkpoint_path, overwrite=True)
+
+    new_partitioner = combined.reference_data._calculate_new_partitions(combined.reference_data.n_partitions())
+
+    info('import_gvs: repartitioning and writing final VDS')
+
+    # add reference base to reference data table as a Hail VDS requirement. Add from FASTA file.
+    rd = add_reference_allele(
+        hl.read_matrix_table(os.path.join(checkpoint_path, 'reference_data'),
+                             _intervals=new_partitioner))
+
+    vd = hl.read_matrix_table(os.path.join(checkpoint_path, 'variant_data'),
+                              _intervals=new_partitioner)
+
+    # read site and vqsr data with same intervals for efficient joins
+    site = hl.read_table(site_path, _intervals=new_partitioner)
+    vqsr = hl.read_table(vqsr_path, _intervals=new_partitioner)
+
+    vd = vd.annotate_rows(filters=hl.coalesce(site[vd.locus].filters, hl.empty_set(hl.tstr)))
+
+    # vqsr ref/alt come in normalized individually, so need to renormalize to the dataset ref allele
+    vd = vd.annotate_rows(as_vqsr = hl.dict(vqsr.index(vd.locus, all_matches=True)
+                                            .map(lambda record: (record.alt + vd.alleles[0][hl.len(record.ref):], record.drop('ref', 'alt')))))
+    vd = vd.annotate_globals(tranche_data=tranche.collect(_localize=False),
+                             truth_sensitivity_snp_threshold=truth_sensitivity_snp_threshold,
+                             truth_sensitivity_indel_threshold=truth_sensitivity_indel_threshold)
+
+    sorted_tranche_data = hl.sorted(vd.tranche_data, key=lambda x: x.truth_sensitivity)
+    vd = vd.annotate_globals(snp_vqslod_threshold=
+                             sorted_tranche_data.filter(lambda x: (x.model == 'SNP') & (
+                                         x.truth_sensitivity >= truth_sensitivity_snp_threshold))
+                             .head().min_vqslod
+                             ,
+                             indel_vqslod_threshold=sorted_tranche_data.filter(lambda x: (x.model == 'INDEL') & (
+                                         x.truth_sensitivity >= truth_sensitivity_indel_threshold))
+                             .head().min_vqslod
+                             )
+
+    is_snp = vd.alleles[1:].map(lambda alt: hl.is_snp(vd.alleles[0], alt))
+    vd = vd.annotate_rows(
+        allele_NO=vd.alleles[1:].map(
+            lambda allele: hl.coalesce(vd.as_vqsr.get(allele).yng_status == 'N', False)),
+        allele_YES=vd.alleles[1:].map(
+            lambda allele: hl.coalesce(vd.as_vqsr.get(allele).yng_status == 'Y', True)),
+        allele_is_snp=is_snp,
+        allele_OK=hl._zip_func(is_snp, vd.alleles[1:],
+                               f=lambda is_snp, alt:
+                               hl.coalesce(vd.as_vqsr.get(alt).vqslod >
```

Comment:
```suggestion
                               hl.coalesce(vd.as_vqsr.get(alt).vqslod >=
```

Reply:
![image](https://user-images.githubusercontent.com/6863459/190492647-33ab32dd-d5fd-49c8-a9dc-28da00cea0f3.png)
https://files.slack.com/files-pri/T0166C33N6A-F042M04CG74/screen_shot_2022-09-15_at_3.27.50_pm.png

Greater than or equal to should pass